src/Okta.DNX.OAuth.ResourceServer/Startup.cs

```diff
@@ -0,0 +1,123 @@
+using Microsoft.IdentityModel.Protocols;
+using System.Security.Claims;
+
+namespace Okta.DNX.OAuth.ResourceServer
+{
+    public class Startup
+    {
+        string clientId = string.Empty;
+        string issuer = string.Empty;
+        string authorizationServerIssuer = string.Empty;
+        string audience = string.Empty;
+
+
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                 .AddJsonFile($"config.{env.EnvironmentName}.json", optional: true)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true);
+
+            if (env.IsEnvironment("Development"))
+            {
+                // This will push telemetry data through Application Insights pipeline faster, allowing you to view results immediately.
+                builder.AddApplicationInsightsSettings(developerMode: true);
+            }
+
+            builder.AddEnvironmentVariables();
+
+            Configuration = builder.Build();
+
+
+            clientId = Configuration["okta:clientId"] as string;
+            issuer = Configuration["okta:organizationUrl"];
+            authorizationServerIssuer = Configuration["okta:authorizationServerIssuer"];
+            audience = Configuration["okta:audience"] as string;
+        }
+
+        public IConfigurationRoot Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add framework services.
+            //services.AddApplicationInsightsTelemetry(Configuration);
+
+            //     var scopePolicy = new AuthorizationPolicyBuilder()
+            //.RequireAuthenticatedUser()
+            //.RequireClaim("scp", "call-api")
+            //.Build();
+
+            services.AddAuthentication();
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy("todo.read",
+                    policy =>
+                    {
+                        policy
+                        .RequireClaim("cid", clientId)
+                       .RequireClaim("http://schemas.microsoft.com/identity/claims/scope", "todolist.read");
+                    }
+                );
+            }
+            );
+
+            services.AddMvc();
+
+            services.AddSingleton<ITodoRepository, TodoRepository>();
+        }
+
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+        {
+            loggerFactory.AddConsole(LogLevel.Trace);
+            loggerFactory.AddDebug();
+
+            //app.UseApplicationInsightsRequestTelemetry();
```

Comment:
Same here - in general, if we don't need it remove it. If you're trying to demonstrate something, you should add a comment like "Uncomment these lines if you want telemetry", etc.